### ISSUE TITLE
Firewall check reports failed if firewalld not installed

### ISSUE THREAD

**Issue by sjjohansen** [cloudera-ps/prereq-checks#56]
The function `_check_service_is_running` gets called from `check_network` to ensure `firewalld` is not running on Centos/RHEL 7. 

```
FAIL  Network: firewalld is not installed
FAIL  Network: firewalld does not auto-start on boot
```

The end goal is we don't want a local firewall running so it either makes sense to not run the `systemctl is-enabled` check if the package is not installed or to enforce installing of `firewalld` and checking it's disabled. I'd say the first option makes more sense.

**Comment by kmizumar** [comment]
This problem is addressed by #58 Fix firewalld service check, I think.

**Comment by kmizumar** [comment]
#58 is merged.
